frontend/src/components/profile/components/profile-info/profile-info.scss

```diff
@@ -1,4 +1,4 @@
-@import '~assets/scss/utils';
+@import 'src/assets/scss/styles';
```

Comment:
check my comments about styles in our team chat. Also if you use imports, please use alias if needed